metrics_config/metrics.go

```diff
@@ -54,7 +58,7 @@ func StartProcessMetrics() {
 		}
 	}()
 	// Short circuit if the metrics system is disabled
-	if !enabled {
+	if !enabled || endpoint == "" {
```

Comment:
This check should be kept to just checking the enabled flag. Still, the empty string is a good thing to check but do it in start.go above.